test/specs/parser/tests/01.parser.js

```diff
@@ -441,6 +441,22 @@ exports.execute = function(options) {
         var baseUri = options.url + "/catalog/" + catalogId + "/entity/" + schemaName + ":" + tableName;
         var facetError = "Given encoded string for facets is not valid.";
         var location, uri;
+        var invalidPageConditionErrorObj = {
```

Comment:
Make these changes to your test cases:

1. Add a test case for Invalid Sort criteria. Sort based on a column that is not sortable. You need to define [column-display](https://github.com/informatics-isi-edu/ermrest/blob/master/user-doc/annotation.md#2016-column-display) annotation for that column with `column_order: false`

2. For invalid page, change the url to have filter. To make sure that you're only removing the page.

3. For invalid filter and facet, add two test cases for each. One with sort/page and one without any modifiers.